src/main/java/com/zipcodewilmington/looplabs/StringDuplicateDeleter.java

```diff
@@ -5,4 +5,59 @@
  * @ATTENTION_TO_STUDENTS You are forbidden from modifying the signature of this class.
  */
 public final class StringDuplicateDeleter extends DuplicateDeleter<String> {
+    public StringDuplicateDeleter(String[] stringArray) {
+        super(stringArray);
+    }
+
+    @Override
+    public String[] removeDuplicates(int maxNumberOfDuplications) {
+        StringBuilder removedString = new StringBuilder();
+        int dupeCount = 0;
```

Comment:
if you just declare dupeCount to 0 inside of the for loop you wouldn't have to reset it to 0 between loops. On top of that you don't seem to use dupeCount outside of the loop so it's outside of the scope it belongs in.